analysis/rogueoutlaw/src/modules/spells/OpportunityDamageTracker.ts

```diff
@@ -3,7 +3,7 @@ import SPELLS from 'common/SPELLS';
 import { FilteredDamageTracker } from '@wowanalyzer/rogue';
 
 class OpportunityDamageTracker extends FilteredDamageTracker {
-  shouldProcessEvent(event) {
+  shouldProcessEvent(event: any): boolean {
```

Comment:
Should be able to use base Event type here instead of `any`

Reply:
Done